### ISSUE TITLE
AODN WAVE NRT - Time not strictly monotonic

### ISSUE THREAD

**Issue by lbesnard** [aodn/content#514]
On pipeline-rc, see the following files:
```
/mnt/ebs/error/AODN_WAVE/DES-QLD_20221029T033000Z_ALBATROSS-BAY_RT_WAVE-PARAMETERS_END-20221029T050000Z.nc
/mnt/ebs/error/AODN_WAVE/DES-QLD_20221030T153000Z_ALBATROSS-BAY_RT_WAVE-PARAMETERS_END-20221030T170000Z.nc
```

This files were downloaded as is from one of the ARDC API, however their time is not strictly monotonic

see for example the time values below where **2022-10-30 16** is repeated twice
```
 TIME = "2022-10-30 15:30", "2022-10-30 16", "2022-10-30 16",
    "2022-10-30 16:30", "2022-10-30 17" ;
}
```

The handler code is capable of dealing with this duplicate value only if all the other variables have an identical value at the same index. However this is not the case as can be seen with the WHTH variable on index [1, 2] which are different:

```
WHTH = 0.2, 0.24, 0.11, 0.27, 0.3 ;
```

So basically, not much we can do about this. These NRT values aren't being ingested and will land in the error directory. Maybe something to contact the partners about?

**Comment by bpasquer** [comment]
The issue is present in the WFS data. All we can do is raise an error when it happens. 
I have contacted the BOM to let them now of the issue.

**Comment by lbesnard** [comment]
fix here: https://github.com/aodn/data-services/pull/1121

**Comment by bpasquer** [comment]
Response from BOM:
>The problem has been identified as an issue with extraction of data from a Bureau database and has been reproduced by technical staff. It seems like the first record should be an hour earlier. Fixing the issue may take some time.

**Comment by bpasquer** [comment]
Update from BOM:
>The issue was found to relate to Daylight Savings time. The problem has been rectified.

**Comment by bpasquer** [comment]
Closing .
Data from Albatross bay is now monotonic
